engine/runtime/src/main/java/org/enso/interpreter/node/callable/IndirectInvokeCallableNode.java

```diff
@@ -90,7 +90,8 @@ Object invokeWithWarnings(
       Warning[] extracted = warnings.getWarnings(warning, null, false);
       return WithWarnings.wrap(EnsoContext.get(this), result, extracted);
     } catch (UnsupportedMessageException e) {
-      throw CompilerDirectives.shouldNotReachHere(e);
```

Comment:
`UnsupportedMessageException` may happen. We are calling to unknown implementation, unknown language and we cannot assume how it is implemented. As such it is safer to _not panic on JVM level_ but propagate such state as a regular _Enso interpreter panic_.

eb5a030a7e does that for whole `engine/runtime` project.